NuRadioReco/modules/channelGalacticNoiseAdder.py

```diff
@@ -361,6 +365,9 @@ def run(
                     + antenna_response['phi'] * channel_noise_spec[2]
                 )
 
+                # scale noise spectrum:
+                channel_noise_spectrum *= self.scaling
+
```

Comment:
@cg-laser are you fine with that? As Philipp wrote in the doc-string this is quite useful for us in SKA but it is somewhat "hacky" which I would like to get your approval.